src/egon/data/datasets/electrical_neighbours.py

```diff
@@ -1640,12 +1650,16 @@ def insert_loads_sq(load_sq=None, scn_name="status2019"):
     """
     sources = config.datasets()["electrical_neighbours"]["sources"]
     targets = config.datasets()["electrical_neighbours"]["targets"]
-
-    ################# TEMPORAL ####################
+    try:
+        load_sq = entsoe_historic_demand()
+    except:
+        logging.warning(
+            """Demand data from entsoe could not be retrieved.
+                        Backup data is used instead"""
+        )
     load_sq = pd.read_csv(
         "data_bundle_powerd_data/entsoe/load_entsoe.csv", index_col="Index"
     )
```

Comment:
These lines now overwrite load_sq always, even if the data was retrieved by the website, right?

Reply:
Right. Indentation error. Fixed in https://github.com/openego/powerd-data/commit/f618dadf663bfdef78e33bd7c43b04f9ad851f50